apps/condo/domains/property/schema/Property.js

```diff
@@ -217,12 +237,27 @@ const Property = new GQLListSchema('Property', {
     hooks: {
         validateInput: ({ resolvedData, existingItem, context, addValidationError }) => {
             if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
-            if (!hasDbFields(['organization', 'type', 'address', 'addressMeta'], resolvedData, existingItem, context, addValidationError)) return
+            if (!hasDbFields(REQUIRED_FIELDS, resolvedData, existingItem, context, addValidationError)) return
+
             const { dv } = resolvedData
+
             if (dv === 1) {
                 // NOTE: version 1 specific translations. Don't optimize this logic
             } else {
-                return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown \`dv\``)
+                return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown 'dv'`)
+            }
+        },
+        afterChange: async ({ operation, existingItem, updatedItem }) => {
+            const isSoftDeleteOperation = operation === 'update' && !existingItem.deletedAt && Boolean(updatedItem.deletedAt)
+            const isCreatedProperty = operation === 'create' && Boolean(updatedItem.address)
+            const isRestoredProperty = operation === 'update' && Boolean(existingItem.deletedAt) && !updatedItem.deletedAt
+            const isAddressUpdated = operation === 'update' && existingItem.address.toLocaleLowerCase('ru') !== updatedItem.address.toLocaleLowerCase('ru')
```

Comment:
why only `ru`?

Reply:
we don't have any locale information here yet. we can try and get organization.country, but it is not clear how to convert it to locale. added TODO (https://doma.atlassian.net/browse/DOMA-1779)